ML/TrainTestMultiClass.py

```diff
@@ -107,13 +107,12 @@ def data_prep(inputCfg, iBin, PtMin, PtMax, OutPutDirPt, DataDf, PromptDf, FDDf)
     return TrainTestData, DataDfPtSel, PromptDfPtSelForEff, FDDfPtSelForEff
 
 
-def train_test(inputCfg, PtMin, PtMax, OutPutDirPt, TrainTestData):
+def train_test(inputCfg, PtMin, PtMax, OutPutDirPt, TrainTestData, HyperPars): #pylint: disable=too-many-statements
```

Comment:
I would pass `iBin` to the function instead of the parameters dictionary

Reply:
Ciao @fcatalan92, done!